enrolment-ui/.env.development

```diff
@@ -0,0 +1 @@
+HOST=local.enrolment.cancercollaboratory.org
```

Comment:
This, to start using dev environment variables, rather than hardwiring things in the manifest